code/game/gamemodes/cult/narsie.dm

```diff
@@ -43,8 +42,22 @@ var/global/list/narsie_list = list()
 /obj/singularity/narsie/large/New()
 	..()
 	if(announce)
-		to_world("<font size='15' color='red'><b>[uppertext(name)] HAS RISEN</b></font>")
+		//to_world("<font size='15' color='red'><b>THE [uppertext(name)] HAS RISEN</b></font>")
 		sound_to(world, sound('sounds/effects/wind/wind_5_1.ogg'))
+		for(var/mob/M in GLOB.player_list) // i believe using this puts the echo effects on the sounds so it doesnt sound like trash.
```

Comment:
There has GOT to be a better way of doing this. Also lol

Reply:
i stole the code from the command announcements code, take it up with the judge :ohyea:

Reply:
this is how the command report code works
man up and deal with it :ohyea: